ImportBlocks/BlockImporter.cs

```diff
@@ -98,25 +147,49 @@ protected override void OnReceive(object message)
                     _blockchainActorRef = startImport.BlockchainActorRef;
                     _doneAction = startImport.OnComplete;
                     blocksBeingImported = GetBlocksFromFile().GetEnumerator();
+                    stateRootsBeingImported = GetStateRootsFromFile().GetEnumerator();
                     // Start the first import
                     Self.Tell(new Blockchain.ImportCompleted());
                     break;
                 case Blockchain.ImportCompleted _:
                     // Import the next batch
-                    List<Block> blocksToImport = new List<Block>();
-                    for (int i = 0; i < BlocksPerBatch; i++)
+                    if (!(blocksBeingImported is null))
```

Comment:
```suggestion
                    if (blocksBeingImported != null)
```
Please, change the block import too